microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -223,33 +248,31 @@ While paused on a line of code, use the **Scope** pane to view and edit the valu
 *  Double-click a property value to change it.
 *  Non-enumerable properties are greyed out.
 
-![The Scope pane](./reference-images/javascript-sources-get-started-js-scope.png)
+![The Scope pane](./reference-images/scope.png)
 
 
 <!-- ====================================================================== -->
 ## Watch the values of JavaScript expressions
 
 Use the **Watch** pane to watch the values of custom expressions.  You can watch any valid JavaScript expression.
 
-![The Watch pane](./reference-images/javascript-sources-get-started-js-watch.png)
+![The Watch pane](./reference-images/watch.png)
 
 *  To create a new watch expression, click the **Add watch expression** (![Add watch expression](./reference-images/add-expression-icon.png)) button.
 
 *  To refresh the values of all existing expressions, click the **Refresh** (![Refresh](./reference-images/refresh-icon.png)) button.  Values automatically refresh while stepping through code.
 
-*  To delete a watch expression, right-click the expression and then select **Delete watch expression** (![Delete watch expression](./reference-images/delete-expression-icon.png)).
+*  To delete a watch expression, right-click the expression and then select **Delete watch expression**.
 
 
 <!-- ====================================================================== -->
 ## View the call stack
 
 While paused on a line of code, use the **Call Stack** pane to view the call stack that got you to this point.
 
-<!--If you are working with async code, check the **Async** checkbox to enable async call stacks.  -->
-
 Click an entry to jump to the line of code where that function was called.  The blue arrow icon represents which function DevTools is currently highlighting.
```

Comment:
```suggestion
Click an entry to jump to the line of code where that function was called.  The blue arrow icon represents which function DevTools is currently highlighting:
```
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/reference?branch=pr-en-us-2894#view-the-call-stack